### PR TITLE
Add two new options

### PR DESCRIPTION
**Pull request by artemryzhov**
1. Folder structure format. If I want to change the structure of folders, I can specify my format, for example: --folder-structure='{:%Y/%m}'

2. Writing exif DateTimeOriginal tag from file creation date, if it's not exists. Sometimes photos are stored in a photostream from different sources (browser, telegram, viber) and they do not have any metadata. Because of this, there is a difficulty in the correct organization of photos on the computer (for example in Shotwell).

### CODE REVIEW

**Comment by louiscaron** [comment]
Nice feature.  I also have made a modification to change the file name and include the folder path.  This helps in finding source of photos.

**Comment by evenwebb** [comment]
Hi there @artemryzhov, I'm getting the following error when using your patch, do you know what might be causing it?

Thank you 😄 

`/it]/usr/local/lib/python2.7/dist-packages/piexif/_load.py:67: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if data[0:2] == b"\xff\xd8":  # JPEG
/usr/local/lib/python2.7/dist-packages/piexif/_insert.py:22: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if image[0:2] == b"\xff\xd8":
`

File: README.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ you can clear a stored password using the `--delete-from-keyring` command-line o
                            [--download-videos]
                            [--auto-delete]
                            [--only-print-filenames]
+                           [--folder-structure=({:%Y/%m/%d})]
+                           [--set-exit-datetime]
```
Comment (evenwebb) [inline README.md]: Should be "exif" not "exit". 😄

**Comment by ndbroadbent** [comment]
Hi @artemryzhov, thanks for this PR, and sorry for the delay! I've rebased and merged your commits.

**Pull request closed without merging**